spec/views/home/service_owner_spec.rb

```diff
@@ -1,7 +1,8 @@
 RSpec.describe "staff/home/service_owner" do
   before do
     assign(:current_user, build(:beis_user))
-    assign(:delivery_partner_organisations, build_list(:delivery_partner_organisation, 2))
+    organisation = build(:delivery_partner_organisation, name: "Partner Org 1")
+    assign(:delivery_partner_organisations, [organisation, build(:delivery_partner_organisation)])
```

Comment:
Is the change here just so you can specify a name for the organisation and test against that on L19?

Reply:
Yep, that's right